lib/validate.js

```diff
@@ -11,7 +11,7 @@ const isValidFn = (opt) => (!opt || typeof opt === 'function' || typeof opt ===
 
 const lessThanOneError = (label) => new Error(`${label} can not be less than 1`)
 const greaterThanZeroError = (label) => new Error(`${label} must be greater than 0`)
-const minIfPresent = (val, min) => val && val < min
+const minIfPresent = (val, min) => val !== null && val < min
```

Comment:
This use to return `0` if `val` was `0` which resulted in a falsy result when it should be true.